helix-term/src/ui/editor.rs

```diff
@@ -341,6 +356,35 @@ impl EditorView {
         }
     }
 
+    pub fn dimmed_view(
+        doc: &Document,
+        anchor: usize,
+        height: u16,
+        theme: &Theme,
+    ) -> Box<dyn Iterator<Item = HighlightEvent>> {
+        let highlight = theme
+            .find_scope_index("ui.virtual.jump.dim")
+            .or_else(|| theme.find_scope_index("comment")); // comment style as fallback
+        if highlight.is_none() {
+            return Box::new(::std::iter::empty());
+        }
+        let highlight = highlight.unwrap();
```

Comment:
```suggestion
        let Some(highlight) = theme
            .find_scope_index("ui.virtual.jump.dim")
            .or_else(|| theme.find_scope_index("comment")) else {
            // comment style as fallback
            return Box::new(::std::iter::empty());
        };
```

Same as above, `let-else` simplifies this.

Reply:
Done as well